.github/workflows/turf.yml

```diff
@@ -15,21 +15,21 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x, 18.x, 20.x]
+        node-version: [18.x, 20.x, 22.x]
 
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
       - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v4
         with:
           node-version: ${{ matrix.node-version }}
 
       - name: Install pnpm
-        uses: pnpm/action-setup@v2
+        uses: pnpm/action-setup@v4
         with:
-          version: 8
+          version: 9
```

Comment:
I would strongly prefer that the build tooling changes go into another PR. Technically removing supported node versions is a break, but also 16 is EOL'd. We might consider just adding 22 and not removing 16 unless we want to 'sneak' that break into 7.x.

Reply:
yeah, makes sense. I just only have pnpm 9 installed on my machine and unfortunately they've dropped support for older node versions

if a maintainer would like to grab any changes you'd like to keep from this PR and send it, please feel free to do so

https://github.com/Turfjs/turf/pull/2623 is already accomplishing most of what I was aiming for, so I'd also be happy to close this one if that's easier